compiler/crates/relay-compiler/src/build_project/artifact_content.rs

```diff
@@ -436,7 +437,7 @@ fn generate_split_operation(
         &mut content,
         "node",
         "NormalizationSplitOperation",
-        &printer.print_operation(schema, normalization_operation),
+        &printer.print_operation(schema, normalization_operation, &project_config.feature_flags.enable_reduced_metadata),
```

Comment:
maybe we can create an instance `printer` with the `enable_reduced_metadata` option, so we don't have to pass it around?